src/theory/bv/bitblast/bitblast_strategies_template.h

```diff
@@ -385,7 +385,7 @@ void DefaultMultBB (TNode node, std::vector<T>& res, TBitblaster<T>* bb) {
     newres.clear(); 
     // constructs a simple shift and add multiplier building the result
     // in res
-    shiftAddMultiplier(res, current, newres);
+    shiftAddMultiplier(node.getNodeManager(), res, current, newres);
```

Comment:
Get this into `NodeManager* nm = ...` outside of the loop

Reply:
Done